engine/tester/config_parse.c

```diff
@@ -2033,10 +2170,30 @@ alloc_and_get_run_item(xmlNodePtr node, tester_cfg *cfg, unsigned int opts,
         *p_run = p;
     }
 
+    /* 'template' is optional */
+    tmpl_name = XML2CHAR(xmlGetProp(node, CONST_CHAR2XML("template")));
+    if (tmpl_name != NULL)
+    {
+        if (opts & TESTER_RUN_ITEM_TEMPLATE)
+        {
+            ERROR("'template' attribute is forbidden in run template");
+            return TE_RC(TE_TESTER, TE_EINVAL);
+        }
+        rc = get_template_by_name(tmpl_name, session, &p->tmpl);
+        if (rc != 0)
+            return rc;
+        free(tmpl_name);
+    }
```

Comment:
IMHO
copy_template_args(p->tmpl, &p->args);
must be done here. Could you explain, what's is wrong if you do this way.

Reply:
If I call this here so it will be done for script and sessions and then I run default_args test (in selftest/ts/tools) than all iterations will be repeated twice. And if I call it here and remove such call from get_session() than some iterations of default_args test will be unhappy: there will be no arguments from session.